lib/utils/url_launcher.dart

```diff
@@ -1,9 +1,11 @@
+import 'package:url_launcher/url_launcher.dart';
 import 'package:url_launcher/url_launcher_string.dart';
 
 void launchURL(String url) async {
-  if (await canLaunchUrlString(url)) {
-    await launchUrlString(url);
+  if (url.startsWith('mailto')) {
+    final url = Uri.parse('mailto:support@circuitverse.org');
+    await launchUrl(url);
   } else {
-    throw 'Could not launch $url';
+    await launchUrlString(url, mode: LaunchMode.externalApplication);
```

Comment:
Only use LaunchMode.externalApplication for this urls
1. https://circuitverse.org/slack
2. https://circuitverse.org/slack
3. https://github.com/CircuitVerse

In any other case open in the app webview itself

Reply:
it's opening app webview for all other links ig, can u please specify which links are opening in external application but need to be opened in app webview